src/Services/DatabaseTools/ORMDatabaseTool.php

```diff
@@ -160,7 +161,11 @@ protected function createDatabaseIfNotExists(): void
         $tmpConnection = DriverManager::getConnection($params);
         $tmpConnection->connect();
 
-        if (!\in_array($dbName, $tmpConnection->getSchemaManager()->listDatabases(), true)) {
+        $databases = $tmpConnection->getDatabasePlatform() instanceof SqlitePlatform
+            ? ['']
```

Comment:
```suggestion
            ? []
```

I think we don't need an empty element.